views/dashboard-application.hbs

```diff
@@ -132,100 +120,71 @@
                                 <option {{#if user.yearOther}} selected{{/if}}>Other</option>
                             </select> </div>
                         <div class='form-group col-md-5'>
-                            <label>GitHub</label>
+                            <label>GITHUB</label>
                             <input class='form-control' type='url' name='github' placeholder='GitHub' value='{{user.github}}'>
                         </div>
 
                         <div class='form-group col-md-5'>
-                            <label>LinkedIn</label>
+                            <label>LINKEDIN</label>
                             <input class='form-control' type='url' name='linkedin' placeholder='LinkedIn' value='{{user.linkedin}}'>
                         </div>
                         <div class='form-group col-md-5'>
-                            <label>Resume</label>
```

Comment:
for each of these, we should apply the capitalization text-transform via css, to avoid potential typos